src/themes/teams/components/Divider/dividerStyles.ts

```diff
@@ -25,13 +25,13 @@ const beforeAndAfter = (size, type, variables): ICSSPseudoElementStyle => ({
 
 const dividerStyles: IComponentPartStylesInput<IDividerPropsWithDefaults, any> = {
   root: ({ props, variables }): ICSSInJSStyle => {
-    const { children, size, type, important, content } = props
+    const { children, fitted, size, type, important, content } = props
     return {
       color: variables.textColor,
       display: 'flex',
       alignItems: 'center',
-      paddingTop: pxToRem(variables.dividerPadding),
```

Comment:
The default padding should already be `0`, so I think you only need to apply padding for the non-fitted case.

```
...(!fitted && {
    paddingTop: pxToRem(variables.dividerPadding),
    paddingBottom: pxToRem(variables.dividerPadding),
}),
```